caam/seco/aes-cbc.c

```diff
@@ -0,0 +1,136 @@
+/* aes-cbc.c
+ *
+ * Copyright (C) 2006-2021 wolfSSL Inc.
+ *
+ * This file is part of wolfSSL.
+ *
+ * wolfSSL is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * wolfSSL is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1335, USA
+ */
+
+#include <wolfssl/options.h>
+#include <wolfssl/wolfcrypt/wc_port.h>
+#include <wolfssl/wolfcrypt/logging.h>
+#include <wolfssl/wolfcrypt/port/caam/wolfcaam.h>
+
+#include <stdio.h>
+
+#define MAX_UPDATES 100
+
+static unsigned int createAesKey()
+{
+    int keyGroup = 1; /* group one was chosen arbitrarily */
+    unsigned int keyIdOut;
+    int keyInfo = CAAM_KEY_PERSISTENT;
+    int keyType = CAAM_KEYTYPE_AES128;
+    byte pubOut[32];
```

Comment:
Is pubOut 32 bytes? 32 = 256-bit key. Consider using `AES_256_KEY_SIZE` or `AES_MAX_KEY_SIZE`?